components/homme/test/benchmarks/NGGPS/coriknl.job

```diff
@@ -0,0 +1,165 @@
+#!/bin/tcsh 
+#
+#  This script will configure, build and run HOMME using the
+#  NGGPS benchmark problem
+#
+#  128 levels
+#  10 tracers
+#    
+#  ne4  ultra-low-res for testing
+#  ne30 (1 degree) 
+#  ne120 (1/4 degree)
+#
+#
+#
+#SBATCH --job-name nggpsbench
+#SBATCH -N 1
+#SBATCH -C knl
+#SBATCH --time=0:15:00
+#XSBATCH --account=FY150001
+#XSBATCH -p ec
+#
+#  set paths to source code, build directory and run directory
+#
+set wdir =  ~/scratch2/knl/nggpsbench           # run directory
+set HOMME = `pwd`/../../..               # /path/to/acme/components/homme
+set MACH = $HOMME/cmake/machineFiles/cori-knl.cmake
+
+#
+#  Which problem?  tiny, ne30 or ne120 configuration
+#
+# use 4 nodes
+set namelist = nggps-tiny.nl ; set name = tiny      
+
+# good Edison nodes:  64  128  256 512 1024 2048  4096  5462
+# elem/node         6144 3072 1536 768  384  192   96    71/72
+#
+# good KNL nodes: 9145  6144  3072 2048 1024 512 256 128 64
+# ele/node        42/43  64   128   192
+# elem/core        0/1    1     2     3    6  12  24  48 96    (  using 64 cores/node )
+#set namelist = nggps-ne256.nl  ; set name = ne256
+
+
+
+    
+#
+#  mpi run command
+#
+setenv OMP_STACKSIZE 16M     #  Cori has 96GB per node. had to lower to 8M on 3K nodes
+setenv PER_NODE  64          #  MPI per node
+setenv OMP_NUM_THREADS 4
```

Comment:
can i change this to 2? to my best knowledge, 2 is always better.

Reply:
yes, I'll change it.